tools/export_torchscript.py

```diff
@@ -76,5 +77,189 @@ def main():
+
+    :param image: Opencv RGB image.
+    :param new_size: Expected (width, height) size after resizing.
+    :param boxes: Optional. If set, also boxes data is rescaled.
+    :return: Resized image or tuple (resized image, resized boxes).
+    """
+    current_width, current_height = get_image_shape(image)
+    final_width, final_height = new_size
+    scale = min(final_width / current_width, final_height / current_height)
+    scaled_width = int(current_width * scale)
+    scaled_height = int(current_height * scale)
+    image = cv2.resize(image, (scaled_width, scaled_height), interpolation=cv2.INTER_CUBIC)
+    if boxes is not None:
+        if len(boxes) > 0:  # Check if array is not empty. This is different situation than boxes = None.
+            boxes[:, :-1] = boxes[:, :-1] * scale
+        return extend_image(image, new_size, boxes)
+    return extend_image(image, new_size)
+
+def extend_image(image: np.ndarray, size: tuple[int, int], boxes: np.ndarray | None = None, dx: int | None = None,
+                 dy: int | None = None) -> np.ndarray | tuple[np.ndarray, np.ndarray]:
+    """
+    Extend image using padding to get expected size. Added areas are filled with gray (128, 128, 128) color.
+    Assume that both current image dimensions are smaller or equal to the expected final size.
+
+    :param image: Opencv RGB image.
+    :param size: Expected (width, height) image size after padding.
+    :param boxes: Optional. Bounding boxes coordinates. Shape (number of boxes, 4).
+    :param dx: Optional. Padding on the left side. If not set, padding is centered.
+    :param dy: Optional. Padding on the top side. If not set, padding is centered.
+    :return: Final image or tuple (final image, final boxes).
+    """
+    current_width, current_height = get_image_shape(image)
+    final_width, final_height = size
+    assert final_width >= current_width
+    assert final_height >= current_height
+    dx = dx or (final_width - current_width) // 2
+    dy = dy or (final_height - current_height) // 2
+    new_image = np.ones((final_height, final_width, 3)) * 128
+    new_image[dy: dy + current_height, dx: dx + current_width] = image
+    if boxes is not None:
+        if len(boxes) > 0:  # Check if array is not empty. This is different situation than boxes = None.
+            boxes[:, 0:-1:2] += dx
+            boxes[:, 1:-1:2] += dy
+        return new_image, boxes
+    return new_image
+
+
+def get_tensor_image(image_path):
+    test_size = (416, 416)
+    img = cv2.imread(image_path)
+    preproc = ValTransform(legacy=False)
+    img, _ = preproc(img, None, test_size)
+    img = torch.from_numpy(img).unsqueeze(0)
+    img = img.float()
+    return img.permute(0, 2, 3, 1)
+
+
+@logger.catch
+def args_main(exp_file, name, opts, decode_in_inference, batch_size, output_name, ckpt=None):
+    exp = get_exp(exp_file, name)
+    exp.merge(opts)
```

Comment:
why opts are merged here and skipped in `export_torchscript.py:19` 
what are opts?